description.md

```diff
@@ -54,6 +54,43 @@ This chapter describes the various technologies used to develop, store, deploy a
 
 ### Third Party (Open) Tools Employed
 
+As part of content development, delivery and consumption, third party tools may be used to enhance the teacher or user experience.
```

Comment:
I would say "will" instead of "may". After all, at Open Education Hub, we focus on technology and tools. I would also somehow word it at "At Open Education Hub".